docs/integrations/data-ingestion/clickpipes/kafka.md

```diff
@@ -179,17 +192,6 @@ Nullable types in Avro are defined by using a Union schema of `(T, null)` or `(n
 
 ClickPipes does not currently support schemas that contain other Avro Unions (this may change in the future with the maturity of the new ClickHouse Variant and JSON data types).  If the Avro schema contains a "non-null" union, ClickPipes will generate an error when attempting to calculate a mapping between the Avro schema and Clickhouse column types.
```

Comment:
This is no longer true.  We now support non-null unions if the variant type is enabled.